guides/release/pages.yml

```diff
@@ -265,6 +265,21 @@
       url: render-performance
     - title: Troubleshooting
       url: troubleshooting
+- title: Accessibility
+  url: accessibility
+  pages:
+    - title: Intro to Accessibility
```

Comment:
Small thing, very optional. "considerations" seems unnecessary in the URL. `accessibility/components` could be nice and allow more for growth.